### PR TITLE
Add licence information

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

File: src/components/molecules/author-information-list/author-information-list.tsx [hunk 1]
```diff
@@ -32,11 +33,12 @@ const AuthorInformation = ({ author }: { author: Author }) => {
   );
 };
 
-export const AuthorInformationList = ({ authors }: { authors: Author[] }) => (
+export const AuthorInformationList = ({ authors, license, year }: { authors: Author[], license?: string, year?: number }) => (
```
Comment (discodavey) [inline src/components/molecules/author-information-list/author-information-list.tsx:36]: Will there be a case of there not being a year or are we assuming if there is not a year then we put the current year?

Comment (discodavey) [reply src/components/molecules/author-information-list/author-information-list.tsx:36]: The only reason is if there is always a year then this should be mandatory

**Comment by elife-alfred-user** [comment]
## Preview environment created!

The preview environment has been deployed to the cluster.

Client URL: https://pr-1175--epp.elifesciences.org
Storybook URL: https://pr-1175--epp-storybook.elifesciences.org

**Note**: If this is a new environment, it can take a few minutes for DNS to update.

**Comment by elife-alfred-user** [comment]
## Preview environment images deployed!

The preview environment has been updated:
client image tag: preview-1175-7d7c046c40a9807e1ab00eabf9e5c841720f84e4-20230927.1411
storybook image tag: preview-1175-7d7c046c40a9807e1ab00eabf9e5c841720f84e4-20230927.1411

Client URL: https://pr-1175--epp.elifesciences.org
Storybook URL: https://pr-1175--epp-storybook.elifesciences.org

**Note**: Updated images can take a few minutes to roll out.

File: src/components/pages/article/tabs/fulltext-tab.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const ArticleFullTextTab = (props: { metaData: MetaData, content: Content
         { props.peerReview && <ReviewContent content={props.peerReview.evaluationSummary.text} isAssessment={true} peerReviewUrl={props.peerReviewUrl}/> }
         <ArticleContent content={props.content} />
         <ReferenceList references={props.metaData.references} />
-        <AuthorInformationList authors={props.metaData.authors}/>
+        <AuthorInformationList authors={props.metaData.authors} license={props.metaData.license} publishedYear={props.metaData.publishedYear} />
```
Comment (scottaubrey) [inline src/components/pages/article/tabs/fulltext-tab.tsx:31]: The publishedYear is probably not the correct year to display for copyright reasons. A version 2 (or version 23) would potential be one of the years previous to when the preprint was published. 

My guess, you'd need to dig into the `ArticleStatus.timeline` but that's just off the top of my head. It's not going to be a pretty solution. Happy to chat it through.

**Review by scottaubrey — CHANGES_REQUESTED** [review]

**Review by scottaubrey — APPROVED** [review]

File: src/components/molecules/content-header/content-header.test.tsx [hunk 3]
```diff
@@ -25,5 +26,7 @@ describe('ContentHeader', () => {
     expect(screen.getByText('X-Men')).toBeInTheDocument();
     expect(screen.getByText('https://doi.org/10.1101/123456')).toBeInTheDocument();
     expect(screen.getByText('title')).toBeInTheDocument();
+    expect(screen.getByText('Copyright information').parentElement).toHaveAttribute('href');
+    expect(screen.getByText('Copyright information').parentElement?.getAttribute('href')).toStrictEqual('#copyright');
```
Comment (discodavey) [inline src/components/molecules/content-header/content-header.test.tsx:30]: Is this the right place to put the copyright as it's content header and there is no copyright here?

Comment (viktoria-mahmud) [reply src/components/molecules/content-header/content-header.test.tsx]: I changed the test to be more generic and now it only checks if the copyright is in the document when it's passed as a parameter.

**Pull request merged**